### ISSUE TITLE
Add documentation page for all folders in a Kirby project

### ISSUE THREAD

**Issue by yandodov** [getkirby/getkirby.com#887]
A [confusion in the forum](https://forum.getkirby.com/t/what-are-the-assets-and-media-folder-used-for/17475) about the `asset()` helper lead me to question what is the purpose of the `assets` folder. As I wrote in the forum post, I think it'd be great if we had an intro page in [the guide](https://getkirby.com/docs/guide) that described what each folder in a Kirby project is used for. Something like:

> - [`content`](https://getkirby.com/docs/guide/content/introduction) is for the TXT files containing the content of your pages, as well as the files accompanying them
> - `site` contains basically all site configuration
>   - [`blueprints`](https://getkirby.com/docs/guide/blueprints/introduction) determine the schema of your page data, as well as metadata for files and users
>   - [`templates`](https://getkirby.com/docs/guide/templates/basics) contains PHP files that output the corresponding HTML for each page type
>   - ...

Basically, you can see a brief overview of how a Kirby project works and use the links to the folders to jump into what you're interested, if you need more details. To me, it would glue the various documentation guides quite nicely and serve as a nice starting point for newcomers.

**Comment by yandodov** [comment]
@texnixe well, I guess you could rewrite the intro of that article to include _all_ folders. I think that the `assets` folder shouldn't be left out as long as Kirby has some special relationship with it. The `assets` root exists, therefore the `assets` folder has a special meaning.

In my opinion, that article could be renamed to something like "Exploring the Roots" where you could use unordered lists to describe what's the purpose of each root. It even semantically makes sense - starting at the _roots_ means really starting at the beginning.

I've used Kirby for about 3 years and I've only now realized that there are `roles` and `collections` roots. I think that each root [here](https://getkirby.com/docs/reference/system/roots) should be in the Guide, even with just a tiny one-liner description of what it does or how it can help you.

Perhaps the Starterkit should also have all roots, each with a small README inside that explains what they do. If you want something without all that noise, you've still got the Plainkit.

**Comment by hiSandog** [comment]
A single canonical project-tree page could distinguish source-controlled inputs (`content`, `site`, `assets`) from runtime/generated directories such as `media`, including whether users may delete or edit each one. Linking every folder name to its deeper guide would keep the overview short while resolving the assets/media confusion. Generating or testing the tree against a fresh starter project could prevent new first-party directories from being omitted later.
